.github/workflows/monitoring.yml

```diff
@@ -0,0 +1,147 @@
+name: Monitoring
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Why do we need this schedule?

Reply:
It's a common pattern in this repository, each submodule is built every night in order to have up-to-date status badges in the README file.